From 470b57d26a5d97939cd51638c38e1d95aa5ec0cf Mon Sep 17 00:00:00 2001
From: Chenhui Zhao <chenhui.zhao@nxp.com>
Date: Thu, 21 Jan 2016 11:50:31 +0800
Subject: [PATCH 0146/1383] powerpc: fix a compile error regarding fsl_guts.h

Got a compile error:
arch/powerpc/sysdev/fsl_rcpm.c:22:26: fatal error: asm/fsl_guts.h: No
such file or directory

The following patch moved arch/powerpc/include/asm/fsl_guts.h to
include/linux/fsl/guts.h, but failed to change the file path in
arch/powerpc/sysdev/fsl_rcpm.c.
commit b97dec3d956a ("powerpc/fsl: Move fsl_guts.h out of arch/powerpc")

Signed-off-by: Chenhui Zhao <chenhui.zhao@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/sysdev/fsl_rcpm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/sysdev/fsl_rcpm.c b/arch/powerpc/sysdev/fsl_rcpm.c
index a653adb..dfc614f 100644
--- a/arch/powerpc/sysdev/fsl_rcpm.c
+++ b/arch/powerpc/sysdev/fsl_rcpm.c
@@ -18,8 +18,8 @@
 #include <linux/of_address.h>
 #include <linux/export.h>
 #include <linux/io.h>
+#include <linux/fsl/guts.h>
 
-#include <asm/fsl_guts.h>
 #include <asm/cputhreads.h>
 #include <asm/fsl_pm.h>
 
-- 
2.8.1

